commit bc31d3b2c7d7f2a03721a05cb3c9a3ce8b1e2e5a
Author: Herbert Xu <herbert@gondor.apana.org.au>
Date:   Mon Oct 8 17:14:34 2007 -0700

    [IPSEC] ah: Remove keys from ah_data structure
    
    The keys are only used during initialisation so we don't need to carry them
    in esp_data.  Since we don't have to allocate them again, there is no need
    to place a limit on the authentication key length anymore.
    
    Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
    Signed-off-by: David S. Miller <davem@davemloft.net>

diff --git a/include/net/ah.h b/include/net/ah.h
index 8f257c159902..5e758c2b5dd5 100644
--- a/include/net/ah.h
+++ b/include/net/ah.h
@@ -9,8 +9,6 @@
 
 struct ah_data
 {
-	u8			*key;
-	int			key_len;
 	u8			*work_icv;
 	int			icv_full_len;
 	int			icv_trunc_len;